spec/services/create_next_edition_service_spec.rb

```diff
@@ -0,0 +1,118 @@
+RSpec.describe CreateNextEditionService do
+  describe ".call" do
+    let(:user) { create :user }
+
+    before do
+      populate_default_government_bulk_data
+    end
+
+    it "aborts if the current edition isn't live" do
+      current_edition = create(:edition, live: false)
+
+      expect { described_class.call(current_edition: current_edition, user: user) }
+        .to raise_error("Can only create a next edition from a live edition")
+    end
+
+    it "updates the documents current edition" do
```

Comment:
this test name doesn't seem to correlate with what's being done. It might be better phrased as `it "returns a new current edition" and then assert that both next edition is current and the previous current edition is not.

Reply:
@kevindew Yes, that makes more sense - thanks.  Will fix.